logzer/logzer_test.go

```diff
@@ -63,8 +63,8 @@ func TestLogFilter(t *testing.T) {
 	log.Info().
 		Str("password", "PASSword").
 		Dict("dict", zerolog.Dict().Str("token", "TOKen")).
-		RawJSON("payload1", []byte(payload1)).
-		RawJSON("payload2", []byte(payload2)).
+		RawJSON("payload1", (payload1)).
+		RawJSON("payload2", (payload2)).
```

Comment:
it looks like parentheses are not needed

Reply:
Done ✓